src/wp-admin/includes/class-wp-debug-data.php

```diff
@@ -1223,6 +1038,196 @@ public static function debug_data() {
 		return $info;
 	}
 
+	/**
+	 * Gets the WordPress server section of the debug data.
+	 *
+	 * @since 6.7.0
+	 *
+	 * @return array
+	 */
+	public static function get_wp_server(): array {
```

Comment:
```suggestion
	public static function get_wp_server() {
```

 I don't think we need strict typing on return values.

Reply:
All previously added functions have a return type, so I'd say we should stick to it and have the return type here.